formaters/csv_formater.py

```diff
@@ -0,0 +1,21 @@
+from metrics.metrics_result import MetricsResult
+
+
+class MetricsCSVFormater:
+    @staticmethod
+    def format(measurement_name: str, metrics: MetricsResult, separator: str = ","):
+        """
+        Format a list of dictionaries with the format key:value
```

Comment:
maybe instead of Format use *Constructs*. Maybe something like that `Constructs a list of dictionaries with the key:value`. However i would rather see a little more clear explanation.